docs/user_guide/index.md

```diff
@@ -0,0 +1,25 @@
+---
+myst:
+  html_meta:
+    "description lang=en": |
+      Documentation for beginner users.
+---
+# User Guide
+
+```{toctree}
+:caption: Installing SED
+:maxdepth: 1
+installation
+```
+
+```{toctree}
+:caption: Basic concepts
+../tutorial/1_binning_fake_data
+../tutorial/8_jittering_tutorial
+../tutorial/3_metadata_collection_and_export_to_NeXus
+```
+
+```{toctree}
+:caption: Advance
+config
```

Comment:
Why is config advanced? This should be in the basic concepts

Reply:
Explanation for Jittering is maybe more something advanced

Reply:
That's true. Was just an initial idea. I think this explanation with each instrument is quite a lot and unnecessary for a new user. But we should definitely include a section to describe creating a new configuration.

And regarding jittering, I thought it's part of the standard workflow so would be good to be in basics, perhaps also a basic intro.